dubbo-cluster/src/test/java/com/alibaba/dubbo/rpc/cluster/loadbalance/LoadBalanceTest.java

```diff
@@ -200,5 +200,33 @@ public void testLoadBalanceWarmup() {
         Assert.assertEquals(100, AbstractLoadBalance
             .calculateWarmupWeight(20 * 60 * 1000, Constants.DEFAULT_WARMUP, Constants.DEFAULT_WEIGHT));
     }
+    
+    public Invocation createInvocation(int i) {
+	    	Invocation invocation = EasyMock.createMock(Invocation.class);
+	    	EasyMock.expect(invocation.getMethodName()).andReturn("method").anyTimes();
```

Comment:
Could you post a new PR while our test framework has changed to Mockito from EasyMock ?